Bullwinkle.class.nut

```diff
@@ -385,7 +395,7 @@ class Bullwinkle {
 
             // Add the retry information
             message.retry <- {
-                "ts": time() + timeout,
+                "ts": ( Bullwinkle._isAgent() ? time() : hardware.micros()/1000000 )  + timeout,
```

Comment:
It seems like to me message.ts should be the eventTime in epoch seconds.  This provides the timestamp for the data and should be what is transmitted to the _partner for it to do whatever useful thing might be done with a timestamp that corresponds to the data.

For this pull request, package._ts is used only for timeouts and calculating latencies.  time() by itself doesn't give very useful info as it is only precise down to the second, hence the need to go to hardware.millis() or hardware.micros().  time() also doesn't tick when there is no RTC present.  Since 36 minutes seems like significant overkill for a timeout or for a network latency, I went with hardware.micros().